stix_shifter_modules/azure_log_analytics/stix_translation/json/to_stix_map.json

```diff
@@ -1,362 +1,1266 @@
+            "object": "ipaddress1"
+          }
+        ],
+        "groupIpReference": {
+          "key": "domain-name.resolves_to_refs",
+          "object": "domain_name",
+          "references": [
+            "ipaddress1"
+          ],
+          "group_ref": true
+        },
+        "groupIpfindingReference": {
+          "key": "x-ibm-finding.ioc_refs",
+          "object": "finding",
+          "references": [
+            "ipaddress1"
+          ],
+          "group_ref": true
+        }
+      }
     },
-    "Description__incident": {
-        "key": "x-azure-security-incident.description",
-        "object": "incident"
+    "url": {
+      "Url": [
+        {
+          "key": "url.value",
+          "object": "url"
+        },
+        {
+          "key": "x-ibm-finding.ioc_refs",
+          "object": "finding",
+          "references": ["url"],
+          "group": true
+        }
+      ]
     },
-    "Labels": {
-        "key": "x-azure-security-incident.labels",
-        "object": "incident"
+    "blob": {
+      "Name": {
+        "key": "x-azure-blob.name",
+        "object": "blob_storage"
+      },
+      "Url": [
+        {
+          "key": "url.value",
+          "object": "blobUrl"
+        },
+        {
+          "key": "x-azure-blob.url_ref",
+          "object": "blob_storage",
+          "references": "blobUrl"
+        },
+        {
+          "key": "x-ibm-finding.ioc_refs",
+          "object": "finding",
+          "references": ["blobUrl"],
+          "group": true
+        }
+      ],
+      "Etag": {
+        "key": "x-azure-blob.etag",
+        "object": "blob_storage"
+      },
+      "BlobContainer": {
+        "Name": {
+          "key": "x-azure-blob.blob_container",
+          "object": "blob_storage"
+        }
+      }
+    },
+    "blob-container": {
+      "Name": {
+        "key": "x-azure-blob.blob_container",
+        "object": "blob_storage"
+      }
+    },
+    "malware": {
+      "Name": {
+        "key": "x-azure-malware.name",
+        "object": "malware"
+      },
+      "Category": {
+        "key": "x-azure-malware.category",
+        "object": "malware"
+      },
+      "Files": {
+        "file": {
+          "Name": {
+            "key": "file.name",
+            "object": "malware_file"
+          },
+          "Directory": [
+            {
+              "key": "directory.path",
+              "object": "dir_obj"
+            },
+            {
+              "key": "file.parent_directory_ref",
+              "object": "malware_file",
+              "references": "dir_obj"
+            }
+          ],
+          "FileHashes": {
+            "SHA256": {
+              "key": "file.hashes.SHA-256",
+              "object": "malware_file"
+            },
+            "SHA1": {
+              "key": "file.hashes.SHA-1",
+              "object": "malware_file"
+            },
+            "MD5": {
+              "key": "file.hashes.MD5",
+              "MD5object": "malware_file"
```

Comment:
I suppose you meant `object` here?

`"MD5object": "malware_file"`  - >   `"object": "malware_file"`

Reply:
> I suppose you meant `object` here?
> 
> `"MD5object": "malware_file"` - > `"object": "malware_file"`

Updated